calico/openstack/mech_calico.py

```diff
@@ -61,6 +63,13 @@
 AGENT_TYPE_FELIX = 'Calico per-host agent (felix)'
 AGENT_ID_FELIX = 'calico-felix'
 
+# Mapping from our endpoint status to neutron's port status.
+PORT_STATUS_MAPPING = {
+    ENDPOINT_STATUS_UP: constants.PORT_STATUS_ACTIVE,
+    ENDPOINT_STATUS_DOWN: constants.PORT_STATUS_DOWN,
+    ENDPOINT_STATUS_ERROR: constants.PORT_STATUS_ERROR,
+}
+
 # The interval between period resyncs, in seconds.
 # TODO: Increase this to a longer interval for product code.
```

Comment:
If we do anything, I think we should introduce the ability to make a dedicated resync process so that we don't share a process with time-critical work.